src/index.js

```diff
@@ -171,6 +176,134 @@ class ReactToPrint extends React.Component {
 
   }
 
+  /**
+   * Handles printing for an array of references
+   * 
+   * @param  {ref[]} refArray 
+   * 
+   */
+  handlePrintMultipleRefs = (refArray) => {
```

Comment:
There's a huge amount of duplicate code between this function and the one above. Can the common parts be extracted into a shared helper?